R/script-to-update-current-chapters.R

```diff
@@ -1,53 +0,0 @@
-#-----------------------------------------------------------#
-# Goal: Update the "Current-Chapters.csv" programmatically
-# Details about this script:
-# 1) Update the meetup url
```

Comment:
@ledell @yabellini unless this is useful for something when monitoring chapters? I thought it was an old script no longer used.

Reply:
I never use this script.

Reply:
I am not sure this script saves any time over making an update manually... I don't think it's needed.